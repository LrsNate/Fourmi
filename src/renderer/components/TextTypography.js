import { withStyles } from "material-ui";
import PropTypes from "prop-types";
import React from "react";

const styles = {
  textTypography: {
    fontFamily: "EB Garamond",
    fontSize: "14pt"
  }
};

const TextTypography = ({ classes, children }) => {
  return <div className={classes.textTypography}>{children}</div>;
};

TextTypography.propTypes = {
  classes: PropTypes.object,
  children: PropTypes.oneOfType([
    PropTypes.node,
    PropTypes.arrayOf(PropTypes.node)
  ]).isRequired
};

TextTypography.defaultProps = {
  classes: {}
};

export default withStyles(styles)(TextTypography);
