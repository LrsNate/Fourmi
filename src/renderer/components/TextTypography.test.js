import { render } from "enzyme";
import React from "react";
import TextTypography from "./TextTypography";

describe("The TextTypography component", () => {
  describe("on an initial load", () => {
    it("matches the snapshot", () => {
      const wrapper = render(<TextTypography>Some text</TextTypography>);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
