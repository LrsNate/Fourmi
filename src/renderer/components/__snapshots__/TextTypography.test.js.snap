// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The TextTypography component on an initial load matches the snapshot 1`] = `
<div
  class="TextTypography-textTypography-1"
>
  Some text
</div>
`;
